tests/unit/service/test_files_service.py

```diff
@@ -337,3 +342,32 @@ async def test_list_all_files_with_timeout(self, file_service: FilesService, mon
         with pytest.raises(TimeoutError):
             async for _ in file_service.list_all(workspace_name="test_workspace", batch_size=10, timeout_s=0):
                 pass
+
+
+@pytest.mark.asyncio
+class TestValidateFilePaths:
+    @pytest.mark.parametrize(
+        "file_paths",
+        [
+            [Path("/home/user/file1.txt"), Path("/home/user/file2.txt")],
+            [Path("/home/user/file1.txt"), Path("/home/user/file1.txt.meta.json")],
+            [Path("/home/user/file1.pdf"), Path("/home/user/file1.pdf.meta.json")],
+        ],
+    )
+    async def test_validate_file_paths(self, file_paths: List[Path]) -> None:
+        FilesService._validate_file_paths(file_paths)
```

Comment:
Same here. private method

Reply:
I think this method can be public

Reply:
I think it's best to keep it private unless we think an SDK consumer wants to use this method. I think it's ok to test private methods in some cases